I have modified the profile page to indicate a clear summary of the number of lists on the platform, number of unfinished and number of finished tasks by the user. Also fixed the background image which was not spanning over the whole page.

BarnaTB opened the pull request:
I have added a background image so that the design is more attractive to the user. Also, I have added a responsive design to the website to give users across all devices a similar view of the app. @Rhytah

BarnaTB committed:
ft(User should be able to create a to-do list and add tasks to it):
Commit delivers a feature that enables a user to create a to-do
list and add tasks to it.
[Deliver #160451803]

BarnaTB committed:
ch(User should be able to navigate the app using a nav bar):
Commit to fix the user navigation bar links.
[Fix #160483756]

BarnaTB committed:
ft(App should be responsive across desktop, laptop and mobile devices):
Commit to deliver a feature for page responsiveness across desktops, laptops and mobile devices.
[Deliver #160537450]

BarnaTB committed:
ft(User should be able to delete a list and view number of tasks created on the app):
Commit to deliver features that enable a user to delete a specific to-do list and also
view a summary of the number of tasks created on the platform
[Deliver #160451839 #160452029]

Rhytah reviewed (commented):
Great job on the navigation, User can fully access different UIs of the app

Rhytah reviewed (changes requested):
implement appending of user input to list for better functionality.
for instance:
<script>
function newElement() {
  var li = document.createElement("LI");
  var inputValue = document.getElementById("todo-input").value;
  var t = document.createTextNode(inputValue);
  li.appendChild(t);
 document.getElementById("todolist").appendChild(li);
  }
  document.getElementById("todo-input").value = "";

</script>

Rhytah reviewed (commented):
The App is responsive

BarnaTB committed:
ft(User profile to view a summary of the number of finished and unfinished tasks, number of lists on the platform):
Commit fixes the issue where the profile page was not objective as to its purpose and also fixes the background-image
which was not spanning over the whole page
[Deliver #160452029]
